deploy/openshift/hawkular-openshift-agent-configmap.yaml

```diff
@@ -11,11 +11,11 @@ data:
     kubernetes:
       tenant: ${POD:namespace_name}
     hawkular_server:
-      url: https://hawkular-metrics.openshift-infra.svc.cluster.local       
+      url: https://hawkular-metrics.openshift-infra.svc.cluster.local
       credentials:
         username: secret:openshift-infra/hawkular-metrics-account/hawkular-metrics.username
         password: secret:openshift-infra/hawkular-metrics-account/hawkular-metrics.password
-      ca_cert_file: secret:openshift-infra/hawkular-metrics-certificate/hawkular-metrics-ca.certificate
+      ca_cert_file: secret:openshift-infra/hawkular-metrics-certs/tls.truststore.crt
```

Comment:
Either we use that, or we build HOSA's own trust store at a separate secret. Using the system's CA isn't enough, as there's an intermediate CA that is unknown to the caller.